test/nbrowser/ProposedChangesPage.ts

```diff
@@ -32,6 +89,8 @@ describe('ProposedChangesPage', function() {
       await driver.find('.action_log_table tr:nth-of-type(2)').getText(),
       '→\ntest1test2\nTEST1TEST2'
     );
+
+    // Check that expanding context works (at least, that it does something).
     assert.equal(await driver.find('.action_log_table button').getText(), '>');
     await driver.find('.action_log_table button').click();
     assert.equal(await driver.find('.action_log_table button').getText(), '<');
```

Comment:
Maybe wrap it oup with `waitToPass`. I also get the `StaleElementReferenceError` here.